db-migration/src/main/resources/db/migration/V2__add_accounts_table.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE roles(
```

Comment:
Czy nie powinno być CREATE TABLE IF NOT EXISTS?

Reply:
Yes

Reply:
Done